### PR TITLE
Update subdomains.json

### PR DESCRIPTION
**Pull request by UnseenAcoustics**

### CODE REVIEW

File: subdomains.json [hunk 1]
```diff
@@ -143,6 +143,7 @@
   "tolis": "tolisdev.xyz",
   "tudor": "artsui.github.io",
   "uki": "uki.vercel.app",
+  "unseen": "unseenacoustics.github.io"
```
Comment (fransallen) [inline subdomains.json:146]: Looks like you missed `,`, correct json format should be `"unseen": "unseenacoustics.github.io",`

**Comment by Arezoomohammadalizadeh** [comment]
🙏

**Comment by Arezoomohammadalizadeh** [comment]
🙏

**Comment by Arezoomohammadalizadeh** [comment]
🙏

**Comment by lakuapik** [comment]
@UnseenAcoustics i've seen you have configured your github pages CNAME to `unseen.is-a.dev` so i will close this. If you still want subdomain thedev.id please re-PR, thanks.

**Pull request closed without merging**